admin_guide/seccomp.adoc

```diff
@@ -0,0 +1,127 @@
+[[admin-guide-seccomp]]
+= Restricting Application Capabilities Using Seccomp
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+[IMPORTANT]
+====
+Seccomp is
+link:https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md[currently
+in alpha support].
+====
+
+Seccomp is used to restrict the set of system calls applications can make,
+allowing administrators greater control over the security of workloads running
+in {product-title}.
+
+Seccomp support is achieved via two annotations in the pod configuration:
+
+* *seccomp.security.alpha.kubernetes.io/pod*: profile applies to all containers in the pod that do not override
+* *container.seccomp.security.alpha.kubernetes.io/<container name>*: container-specific profile override
```

Comment:
@pweil- I'm not sure what the above are. Why are we configuring pods here? Is this something you have to configure before you can go through the tasks below?